app/models/post.rb

```diff
@@ -2,15 +2,17 @@ class Post < ApplicationRecord
   has_many :likes, foreign_key: 'posts_id'
   has_many :comments, foreign_key: 'posts_id'
   belongs_to :author, class_name: 'User', dependent: :destroy, foreign_key: 'author_id'
-  after_save :post_counter
+  after_save :update_post_counter
 
-  def latest_comments
-    comments.order(created_at: :desc).limit(5)
-  end
+  validates :title, presence: true, length: { in: 1..250 }
+  validates :author_id, presence: true, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validates :likes_counter, presence: true, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
```

Comment:
- [ ] Great job so far, however, you should not delete the  `:comments_counter` validations since this is among the project requirements. Kindly bring it back ✍🏻

Reply:
`:comment_counter` validations is added back. 😄